apps/files_sharing/src/components/NewFileRequestDialog/FileRequestFinish.vue

```diff
@@ -0,0 +1,219 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<div>
+		<!-- Request note -->
+		<NcNoteCard type="success">
+			{{ t('files_sharing', 'Once created, you can share the link below to allow people to upload files to your directory.') }}
+		</NcNoteCard>
+
+		<!-- Copy share link -->
+		<NcInputField ref="clipboard"
+			:value="shareLink"
+			:label="t('files_sharing', 'Share link')"
+			:readonly="true"
+			:show-trailing-button="true"
+			:trailing-button-label="t('files_sharing', 'Copy to clipboard')"
+			@click="copyShareLink"
+			@click-trailing-button="copyShareLink">
+			<template #trailing-button-icon>
+				<IconCheck v-if="isCopied" :size="20" @click="isCopied = false" />
+				<IconClipboard v-else :size="20" @click="copyShareLink" />
```

Comment:
Having click listeners on non-interactive elements (icons here) is bad and will cause accessibility warning / maybe even errors.

Instead just use `click-trailing-button` which you already do for the clipboard one, so just adjust the function (see below).

```suggestion
				<IconCheck v-if="isCopied" :size="20" />
				<IconClipboard v-else :size="20" />
```

Reply:
Yeah, I did this before and the click did not work either. That's why I tried to move it to the icon element. That is temporary :+1: 
I need to double check again